spec/requests/posts_spec.rb

```diff
@@ -2,16 +2,34 @@
 
```

Comment:
Has `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory 🐐 Well done!